torch/_decomp/__init__.py

```diff
@@ -211,6 +211,8 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.elu_backward,
             aten._embedding_bag,
             aten.embedding_dense_backward,
+            aten.empty,
```

Comment:
aten.empty.memory_format has been marked as core aten. 
Don't need to decomp this. 

See the `main` (latest) version of the IR. 
https://pytorch.org/docs/main/ir.html